### PR TITLE
feat: retire diamond_collector-summary Unleash flag

### PR DESCRIPTION
**Pull request by mzikherman**
Now that we launched, we can remove the usage here in Force.

### CODE REVIEW

File: src/Components/Inquiry/Views/InquiryInquiry.tsx [hunk 1]
```diff
@@ -147,55 +135,39 @@ const InquiryInquiry: React.FC<InquiryInquiryProps> = ({ artwork }) => {
           </Text>
         </Box>
       </Flex>
-
       <Separator my={2} />
-
```
Comment (dzucconi) [inline src/Components/Inquiry/Views/InquiryInquiry.tsx:152]: Someone doesn't like whitespace

Comment (mzikherman) [reply src/Components/Inquiry/Views/InquiryInquiry.tsx:152]: The editor auto-did this on save!

Comment (dzucconi) [reply src/Components/Inquiry/Views/InquiryInquiry.tsx:152]: Strange!

**Review by iskounen — APPROVED** [review]
LGTM 👍

**Pull request merged**